fxa-auth-db-server/index.js

```diff
@@ -120,6 +120,7 @@ function createServer(db) {
   api.get('/sessionToken/:id/verified', withIdAndBody(db.sessionTokenWithVerificationStatus))
   api.get('/keyFetchToken/:id/verified', withIdAndBody(db.keyFetchTokenWithVerificationStatus))
   api.post('/tokens/:id/verify', withIdAndBody(db.verifyTokens))
+  api.post('/tokens/:shortCode/verifyShortCode', op(req => db.verifyTokensShortCode(req.params.shortCode, { uid: req.body.uid } )))
```

Comment:
Is the `withIdAndBody` helper not usable here?  It seems a little odd to have this route as the only one that's not using the helper.

Reply:
Yea it wasn't useable. Using that would create a buffer instead of sending the string. This was based off how `signinCodes` [do it](https://github.com/mozilla/fxa-auth-db-mysql/blob/6851f4e18bc68c8beb00c910bf7391664aef9c3a/fxa-auth-db-server/index.js#L211-L211).